app/lib/methods/subscriptions/room.js

```diff
@@ -80,6 +80,7 @@ export default class RoomSubscription {
 	handleConnection = () => {
 		reduxStore.dispatch(clearUserTyping());
 		RocketChat.loadMissedMessages({ rid: this.rid }).catch(e => console.log(e));
+		RocketChat.subscriptionRead(this.rid);
```

Comment:
- This is a call to a REST API, so it might throw an error. You're not handling it, so it might crash the app
- We should only call it after `loadMissedMessages` is resolved, otherwise we would be marking as read messages the user never downloaded.